v2/passwordless/pre-built-ui/further-reading/passwordless-login.mdx

```diff
@@ -91,11 +91,7 @@ Entering an incorrect OTP too many times results in the user being navigated bac
 
 ### Logging in via OTP and Magic link simultaneously 
 
-There is a edge case wherein the end user get's both an OTP and a magic link, and whilst viewing the enter OTP screen, then also click on the magic link separately. The magic link click will open a new tab and will consume the link to log the user in. 
-
-Since the user is logged in, the enter OTP screen will change to show the UI as shown below
-
-<img src="/img/otp-magic-link-same-time.png" alt="Magic link and OTP at the same time" />
+There is a edge case wherein the end user get's both an OTP and a magic link, and whilst viewing the enter OTP screen, then also click on the magic link separately. The magic link click will open a new tab and will consume the link to log the user in. The enter OTP screen will continue to show the enter OTP UI until user refreshes the page.
```

Comment:
```suggestion
There is a edge case wherein the end user get's both an OTP and a magic link, and whilst viewing the enter OTP screen, then also click on the magic link separately. The magic link click will open a new tab and will consume the link to log the user in. The enter OTP screen will continue to show the enter OTP UI until user refreshes the page, after which, it will redirect to the post login screen.
```